LiskHQ/lisk
# API performance benchmarking via newrelic - Closes #1623, #1780

### What was the problem?

There were no tool or strategy available to benchmark/measure the performance of different aspects of application. 

### How did I fix it?

Integrated newrelic with custom code instrumentation to monitor the application performance. 

### How to test it?

Set `NEW_RELIC_LICENSE_KEY` variable and start the app normally. Then check the numbers in your application under newrelic account.

### Review checklist

* The PR resolves #1623, #1780
* All new code is covered with unit tests
* All new code was formatted with Prettier
* Linting passes
* Tests pass
* Commit messages follow the [commit guidelines](CONTRIBUTING.md#git-commit-messages)
* Documentation has been added/updated
